### ISSUE TITLE
RFE: wgrib2 doesn't have IPOLATES (interpolation library) support

### ISSUE THREAD

**Issue by akrherz** [conda-forge/wgrib2-feedstock#11]
Issue: attempting to use wgrib2=2.0.7 h547465b_1003 on linux64 python=3.8 fails with IPOLATES package is not installed

Running a command, like so, will fail:
```
 wgrib2 /tmp/tmpo0yl3_jv.grib2 -set_grib_type same -new_grid_winds earth -new_grid latlon -126:3050:0.02 23.01:1340:0.02 /tmp/tmpy9u4l5h7.grib2
```
with the error
```
IPOLATES package is not installed
```

So it would be nice is `USE_IPOLATES=1` was enabled in the build, but have no idea how difficult this would be :)  I figured I'd file the issue and try to generate a PR  to do this.  Thanks.

**Comment by ocefpaf** [comment]
I'm updating it to `2.0.8` and it looks like the option `USE_IPOLATES=3` is the default.

Can you give me a small file that we could use in the tests?

**Comment by akrherz** [comment]
Thanks @ocefpaf , here's [an example](https://mesonet.agron.iastate.edu/pickup/example.grib2) file used in the example command in my initial message.

**Comment by ocefpaf** [comment]
BTW, the current version does have `USE_IPOLATES=1`, so something is failing in the build. Not sure if that is related to the `no_proj.patch `we have.

**Comment by akrherz** [comment]
Looking int this, oh dear, this involves building an ancient [iplib](https://www.nco.ncep.noaa.gov/pmb/docs/libs/iplib/iplib.tar) with no GNU makefile and perhaps an older Proj too.  This appears to be a bridge too far :(  Thanks for attempting this ocefpaf!  Will see about a "clean" 2.0.8 update now.

**Comment by ocefpaf** [comment]
Ouch! That is not worth the effort. Sorry!

**Comment by akrherz** [comment]
A gentle cc for @opoplawski to see if he has any comments on the wgrib2 and iplib combo.  TIA.

**Comment by opoplawski** [comment]
Yeah, the code is in pretty rough shape.  I had the same request for the Fedora package (https://bugzilla.redhat.com/show_bug.cgi?id=1798251) - but I'm at least waiting for someone to respond to https://github.com/NOAA-EMC/NCEPLIBS-ip2/issues/2.

**Comment by opoplawski** [comment]
Yeah, looks like this requires the use of proj 4.8 which is quite old.  Not sure I'll pursue this further.

**Comment by ocefpaf** [comment]
> Yeah, looks like this requires the use of proj 4.8 which is quite old. Not sure I'll pursue this further.

This would be an herculean job for us in `conda-forge` and the resulting package would not be compatible with most of the stack. I don't think it is worth for us too.
